packages/expo-yarn-workspaces/index.js

```diff
@@ -74,6 +74,9 @@ exports.createMetroConfiguration = function createMetroConfiguration(projectPath
       // test-suite includes a db asset
       assetExts: [...assetExts, 'db'],
 
+      // Include .cjs and .mjs files
+      sourceExts: [...sourceExts, 'cjs', 'mjs'],
```

Comment:
Do we need mjs?

Reply:
Possibly if a module publishes only ESM and uses .mjs extensions (instead of "type": "module" in package.json).

Reply:
I guess it's fine but two considerations:
- this doesn't support `native.mjs`, `native.cjs`, android, ios, etc. 
- increases allowable project complexity.

Reply:
I don't object that much to backing out .mjs. The downside is that we don't have a need for it right now. The upside is that I currently don't see a reason why we'd support .cjs but not .mjs.

I don't have a lot of thoughts around the subextensions other than that constraints are liberating and by not supporting them with .cjs and .mjs, we don't make it harder to move to a subextension-free future with the greater bundler/JS ecosystem that doesn't use subextensions.